test/utilities.js

```diff
@@ -543,6 +543,53 @@ describe('utilities', function () {
         }
       }
     });
+
+    it('should return new assertion with flags copied over', function() {
+      var assertionConstructor;
+
+      chai.use(function (_chai, utils) {
+        assertionConstructor = _chai.Assertion;
+
+        _chai.Assertion.overwriteProperty('and', function (_super) {
```

Comment:
I think we need to create a custom property to overwrite instead of using `and` since this change will bleed over into subsequent tests.